features/polyExplorer/src/components/entityList/entityList.jsx

```diff
@@ -24,13 +24,16 @@ function EntityList({ entities, showGrouped, sideLabel, expand }) {
 
     function Content({ children }) {
         if (expand) return <>{children}</>;
+
+        const marginStyles = { margin: "40px 0 0 0" };
         return (
             <InfiniteScroll
                 dataLength={allGroups.length - groupsToLoad.length}
                 next={handleLoadMoreData}
                 scrollThreshold="80%"
                 hasMore={hasMore}
                 scrollableTarget="entity-list"
+                style={marginStyles}
```

Comment:
@RichardSto just wanted to know, if that inline style is okay here?

Reply:
or @fhd ?

Reply:
Should be fine!

Reply:
Preferably use the css selector of the Infinitescroll though